File: ruan2/Problema0109/Problema0109.py
#Crie uma calculadora capaz de somar, subtrarir, multiplicar e dividir 2 números. 
#O usuário deve informar a operação desejada e os valores a serem calculados

print("Escolha uma das operações a baixo (1, 2, 3 ou 4)")
print("1 - Adição")
print("2 - Subtração")
print("3 - Multiplicação")
print("4 - Divisão")
op = int(input("Digite a sua opção: "))

if(op > 4 or op < 1):
    print("Digite um valor de operação válido")
else:
    num1 = float(input("Digite o valor do primeiro numero: "))
    num2 = float(input("Digite o valor do segundo numero: "))
    if(op == 1):
        valFinal = num1 + num2
    elif(op == 2):
        valFinal = num1 - num2
    elif(op == 3):
        valFinal = num1 * num2
    else:
        valFinal = num1 / num2

print("Valor final:",valFinal)

File: ruan2/Problema0109/Problema0109.js
const prompt = require("prompt-sync")({sigint: true})

/*Crie uma calculadora capaz de somar, subtrarir, multiplicar e dividir 2 números. 
O usuário deve informar a operação desejada e os valores a serem calculados */

var op
var num1, num2
var valFinal

console.log("Escolha uma das operações a baixo (1, 2, 3 ou 4)")
console.log("1 - Adição")
console.log("2 - Subtração")
console.log("3 - Multiplicação")
console.log("4 - Divisão")
op = Number(prompt("Digite a sua opção: "))

if(op > 4 || op < 1){
    console.log("Digite um valor de operação válido")
}
else{
    num1 = Number(prompt("Digite o valor do primeiro numero: "))
    num2 = Number(prompt("Digite o valor do segundo numero: "))

    if(op == 1){
        valFinal = num1 + num2
    }
    else{
        if(op == 2){
            valFinal = num1 - num2
        }
        else{
            if(op == 3){
                valFinal = num1 * num2
            }
            else{
                valFinal = num1 / num2
            }
        }
    }
}

console.log("Valor final: "+valFinal)



